pkg/cable/libreswan/libreswan.go

```diff
@@ -46,29 +47,26 @@ func init() {
 }
 
 type libreswan struct {
-	secretKey string
-
-	debug   bool
-	logFile string
-
 	localEndpoint types.SubmarinerEndpoint
```

Comment:
The reorder of parameters here was enforced by golangci-lint "malign" checker.

It was complaining because if I reorder the struct fields it will need 256bytes instead of 264 for some reason. We don't care much about optimization here, but golangci-lint wouldn't pass otherwise.